examples/src/app/forms/universal-validator/universal-validator.component.html

```diff
@@ -1,42 +1,42 @@
 <form>
-    <md-card class="card">
-        <md-card-header>
-            <md-card-title>Universal</md-card-title>
-        </md-card-header>
-        <md-card-content>
-            <md-input-container>
-                <input mdInput type="text" [(ngModel)]="model.whitespace" name="whitespace" #formControl="ngModel" noWhitespace placeholder="Whitespace">
-                <md-hint *ngIf="formControl.hasError('noWhitespaceRequired')" class="error">No whitespace allowed</md-hint>
-            </md-input-container>
+    <mat-card class="card">
+        <mat-card-header>
+            <mat-card-title>Universal</mat-card-title>
+        </mat-card-header>
+        <mat-card-content>
```

Comment:
Can you make that the example under each other, like the reactive-forms

Reply:
done